tests/ui/dyn-keyword/trait-missing-dyn-in-qualified-path.edition2021.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0782]: trait objects must include the `dyn` keyword
+  --> $DIR/trait-missing-dyn-in-qualified-path.rs:5:19
+   |
+LL |     let x: u32 = <Default>::default();
+   |                   ^^^^^^^
+   |
+help: add `dyn` keyword before this trait
```

Comment:
Right, so I guess we should add `if this trait is object-safe` until we check for object-safety again(?)